premailer/__main__.py

```diff
@@ -114,6 +115,11 @@ def main(args):
             options.disable_basic_attributes.split()
         )
 
+    if options.outfile is not sys.stdout:
+        options.outfile.close()
+        options.outfile = codecs.open(options.outfile.name, 'w',
```

Comment:
Can you indent this like this:
```python
options.outfile = codecs.open(
    options.outfile.name, 'w', encoding='UTF-8'
)
```
That way the second lines massive line of spaces isn't dependent on the length of the first line.